src/frontend/web_application/src/components/ContactProfile/components/ContactProfileForm/index.jsx

```diff
@@ -73,6 +73,10 @@ class ContactProfileForm extends Component {
 
   render() {
     const { __ } = this.props;
+    const dateProps = {};
+    if (this.state.contact.infos.birthday) {
+      dateProps.selected = new Date(this.state.contact.infos.birthday);
```

Comment:
`this.state.contact.infos.birthday` should probably remain a string (whether ISO or not) all along, and only be `moment`-ed when passed to `DatePicker` component here https://github.com/CaliOpen/Caliopen/pull/401/files#diff-140efc3fd7e83b80942d6152cc494913R47

see https://github.com/CaliOpen/Caliopen/pull/401/files#r128467390